src/Symfony/Component/Translation/Tests/DependencyInjection/TranslationPassTest.php

```diff
@@ -30,21 +30,21 @@ public function testValidCollector()
 
         $container = new ContainerBuilder();
         $container->setDefinition('translator.default', $translator);
-        $container->setDefinition('translation.loader', $loader);
+        $container->setDefinition('translation.reader', $loader);
 
         $pass = new TranslatorPass();
         $pass->process($container);
 
         $expected = (new Definition())
             ->addTag('translation.loader', array('alias' => 'xliff', 'legacy-alias' => 'xlf'))
-            ->addMethodCall('addLoader', array('xliff', new Reference('translation.loader')))
-            ->addMethodCall('addLoader', array('xlf', new Reference('translation.loader')))
+            ->addMethodCall('addLoader', array('xliff', new Reference('translation.reader')))
+            ->addMethodCall('addLoader', array('xlf', new Reference('translation.reader')))
         ;
         $this->assertEquals($expected, $loader);
 
-        $this->assertSame(array('translation.loader' => array('xliff', 'xlf')), $translator->getArgument(3));
+        $this->assertSame(array('translation.reader' => array('xliff', 'xlf')), $translator->getArgument(3));
 
-        $expected = array('translation.loader' => new ServiceClosureArgument(new Reference('translation.loader')));
+        $expected = array('translation.reader' => new ServiceClosureArgument(new Reference('translation.reader')));
```

Comment:
hmm, we can't make it an alias as it has a different API. It must still be wired explicitly, and we'll remove the code when removing the service.